va/drm/va_drm_utils.c

```diff
@@ -110,6 +112,16 @@ VA_DRM_GetDriverName(VADriverContextP ctx, char **driver_name_ptr, int candidate
         return VA_STATUS_ERROR_INVALID_DISPLAY;
 
     *driver_name_ptr = va_DRM_GetDrmDriverName(drm_state->fd);
+
+    /* Map vgem to WSL2 for Windows subsystem for linux */
+    struct utsname sysinfo = {};
+    if (!strncmp(*driver_name_ptr, "vgem", 4)
+        && (uname(&sysinfo) >= 0)
+        && strstr(sysinfo.release, "WSL")) {
```

Comment:
nit:
```
    if (!strncmp(*driver_name_ptr, "vgem", 4) && uname(&sysinfo) >= 0 &&
        strstr(sysinfo.release, "WSL")) {
...
```

But more importantly, can we have this combination (Windows/WSL) in the CI matrix.

Reply:
> can we have this combination (Windows/WSL) in the CI matrix

Can you elaborate what do you mean here? which CI matrix? on which side?

Reply:
WSL runs a virtualized Linux distro on top of Windows like [Ubuntu WSL](https://ubuntu.com/wsl) using the original linux libva binaries, no special build is necessary.

Reply:
Was under the impression that everything WSL is built within a different environment. Glad to hear that's not the case :+1: